src/main/java/com/epam/emotionalhelp/controller/QuizController.java

```diff
@@ -0,0 +1,72 @@
+package com.epam.emotionalhelp.controller;
+
+import com.epam.emotionalhelp.controller.config.CORSConfig;
+import com.epam.emotionalhelp.controller.dto.QuizRequestDto;
+import com.epam.emotionalhelp.controller.response.ResponseHandler;
+import com.epam.emotionalhelp.controller.response.ResponseMessage;
+import com.epam.emotionalhelp.service.QuizService;
+import com.epam.emotionalhelp.service.mapper.QuizMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static com.epam.emotionalhelp.controller.util.EndpointName.QUIZ;
+
+@RestController
+@RequestMapping(path = QUIZ)
+@CrossOrigin(origins = CORSConfig.LOCALHOST)
+@RequiredArgsConstructor
+public class QuizController {
+    private final QuizService quizService;
+    @GetMapping
+    public ResponseEntity<Object> findAll(Pageable pageable) {
+        var quizzes = quizService.findAll(pageable);
+        return ResponseHandler.generateResponse(ResponseMessage.SUCCESSFULLY_RECEIVED, HttpStatus.OK,
+                QuizMapper.pageEntityToPageDto(quizzes));
```

Comment:
We've agreed to move mapping logic to service layer, haven't we? No need to additional complexity at API level..